Lib/test/test_pkgutil.py

```diff
@@ -92,6 +93,15 @@ def test_getdata_zipfile(self):
 
         del sys.modules[pkg]
 
+    def test_iter_modules(self):
+        #see: issue44061
+        actual = list(pkgutil.iter_modules([Path("/home")], 'somepackage.somesubpackage'))
+        self.assertListEqual([], actual)
```

Comment:
Can we have a test that _finds_ something? This one could be satisfied by ignoring `Path` objects, which isn't the fix we've implemented.

Reply:
Sure, just pushed a change, can you please check again?